northstar/src/runtime/mount.rs

```diff
@@ -166,19 +162,27 @@ async fn mount(
     fd: RawFd,
     fsimg_offset: u64,
     fsimg_size: u64,
-    name: &Name,
     version: &Version,
     verity_header: Option<VerityHeader>,
     hashes: Option<Hashes>,
     target: &Path,
     verity: bool,
 ) -> Result<MountInfo, Error> {
-    let dm_name = format!("northstar_{}_{}_{}", process::id(), name, version);
-
     // Acquire a loop device and attach the backing file. This operation is racy because
     // getting the next free index and attaching is not atomic. Retry the operation in a
     // loop until successful or timeout.
     let start = time::Instant::now();
+
+    if !target.exists() {
+        debug!("Creating mount point {}", target.display());
+        std::fs::create_dir_all(&target).map_err(|e| {
+            Error::Io(
+                format!("Failed to create directory {}", target.display()),
+                e,
+            )
+        })?;
```

Comment:
Maybe we should try to implement that `Context` trait for these mount results too

Reply:
Or even better get rid of `runtime::mount::Error` and use `runtime::error::Error`

Reply:
yes! that would be great!